src/common/DepositPage/index.tsx

```diff
@@ -48,6 +54,35 @@ export const DepositPage = ({
           <Checkbox onChange={handleOptInFragmentationChange} className="dm-sans dm-sans__x mt-4">
             Advanced: split deposit into multiple fragments for concurrent trade support
           </Checkbox>
+          <div className="mt-4">
+            <Title className="dm-sans dm-sans__x dm-sans__bold dm-sans__grey" level={3}>
+              Previous addresses
+            </Title>
+            <div className="panel panel__grey scrollbar" style={{ maxHeight: '300px', overflowY: 'scroll' }}>
+              {depositAddresses
+                ?.slice()
```

Comment:
Show the text "You generated addresses will appear here" if we are at the beginning and there are no previous deposit addresses